docs/components/modeler/desktop-modeler/index.md

```diff
@@ -0,0 +1,34 @@
+---
+id: index
+title: Desktop Modeler
+sidebar_label: About
+description: "Camunda Desktop Modeler is a desktop app for modeling BPMN, DMN, and Forms, compatible with Camunda Platform 7 and Camunda Platform 8."
+---
+
+<span class="badge badge--cloud">Camunda Platform 7 and 8</span>
+
+Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms and supports you in building executable diagrams with Camunda.
+
+![Desktop Modeler Screenshot](./img/new-diagram.png)
+
+## Features
+
+- Design BPMN, DMN, and forms
+- Implement Camunda 7 and Camunda 8 process applications
```

Comment:
Nitpicky, but we do need to spell out Camunda Platform 7 and Camunda Platform 8.